scripts/ci/build-and-test.sh

```diff
@@ -10,20 +10,29 @@ cd $(dirname $0)/../..
 source scripts/ci/sources/mode.sh
 source scripts/ci/sources/tunnel.sh
 
+# Check if tests can be skipped
+skip_tests=true
+for filename in $(git diff --name-only $TRAVIS_BRANCH...HEAD); do
```

Comment:
I didn't talk about PRs. Good to see that you tested it for PRs.

Can you try merging your PR on your fork and see if everything works there too?